04 Implementation/src/entities/AfsaetningImpl.java

```diff
@@ -2,7 +2,7 @@
 
 import entities.exceptions.NegativAntalException;
 
-public class AfsaetningImpl implements Afsaetning, Observable {
+public class AfsaetningImpl implements Afsaetning {
     private int antal;
     private ObserverManager observerManager;
```

Comment:
```suggestion
    private int antal = 0;
    private ObserverManager observerManager;
```
Benyt eksplicit initialisering af field.